templates/image-builder.yml

```diff
@@ -77,11 +77,31 @@ objects:
               value: "${CW_LOG_GROUP}"
             - name: CW_AWS_REGION
               value: "${CW_AWS_REGION}"
-            - name: EXAMPLE
+            - name: PGHOST
               valueFrom:
-                configMapKeyRef:
-                  key: example
-                  name: image-builder-config
+                secretKeyRef:
+                  name: db
+                  key: host
```

Comment:
So I think the `name` should be `image-builder-db` and the key should be `db.host`, `db.port`, etc.

The name of the secret itself is `image-builder-db`. See https://gitlab.cee.redhat.com/service/app-interface#manage-rds-databases-via-app-interface-openshiftnamespace-1yml

Reply:
Thanks! this makes more sense to me know.